frontend/src/app/projects/create-project/create-project.stories.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+import { userEvent, within } from '@storybook/test';
+import {
+  mockProject,
+  MockProjectWrapperService,
+} from '../../../storybook/project';
+import {
+  mockProjectUsers,
+  MockProjectUserService,
+} from '../../../storybook/project-users';
+import { ProjectUserService } from '../project-detail/project-users/service/project-user.service';
+import { ProjectWrapperService } from '../service/project.service';
+import { CreateProjectComponent } from './create-project.component';
+
+const meta: Meta<CreateProjectComponent> = {
+  title: 'Project Components / Create Project',
```

Comment:
```suggestion
  title: 'Project Components/Create Project',
```

Otherwise the files generated by storycap have whitespaces in it.